.env.sample

```diff
@@ -10,23 +10,30 @@ PORT_L2_EXECUTION_ENGINE_P2P=30306
 PORT_PROVER_SERVER=9876
 PORT_PROMETHEUS=9091
 PORT_GRAFANA=3001
-# Comma-separated profiles to be used by default (l2_execution_engine,proposer,prover)
+# Comma-separated profiles to be used by default (l2_execution_engine,prover)
+# Proposer is deprecated for community as of Based Preconfs, so profile has been removed.
 COMPOSE_PROFILES=l2_execution_engine
 
 # Comma separated L2 execution engine bootnode URLs for P2P discovery bootstrap
 BOOT_NODES=enode://266a8e3b5e44201eca9c368d58aa59a7750295397e77d5b32aea2644f9962cbc4e1cb0543aab0480995a209408174413f65e5ce253d60bb83d22d3b8ab12eb89@34.142.239.251:30303,enode://264a7fc4bd1ee16cfc6eb420c643407bfc61b9c9534c5a39ba6e68c8759beda2fbeccefee8677385e3d99691eeb218da4bce7f5207cf38594ac0f6a53c128b9b@35.247.159.156:30303,enode://2d4e5b7ec0c57f9def6ebe72f9bd1f65c33c87b7dc38875bbb147c10e8ec9a8cd157558b695f9a02ac6ad789f300fab4f1f19d41273956491372e96880a3459f@34.126.90.255:30303,enode://57f4b29cd8b59dc8db74be51eedc6425df2a6265fad680c843be113232bbe632933541678783c2a5759d65eac2e2241c45a34e1c36254bccfe7f72e52707e561@104.197.107.1:30303,enode://87a68eef46cc1fe862becef1185ac969dfbcc050d9304f6be21599bfdcb45a0eb9235d3742776bc4528ac3ab631eba6816e9b47f6ee7a78cc5fcaeb10cd32574@35.232.246.122:30303
 
 # Taiko protocol contract addresses
-TAIKO_L1_ADDRESS=0x06a9Ab27c7e2255df1815E6CC0168d7755Feb19a
+TAIKO_INBOX_ADDRESS=0x06a9Ab27c7e2255df1815E6CC0168d7755Feb19a
 TAIKO_TOKEN_L1_ADDRESS=0x10dea67478c5F8C5E2D90e5E9B26dBe60c54d800
-TAIKO_L2_ADDRESS=0x1670000000000000000000000000000000010001
-TAIKO_WRAPPER=0x9F9D2fC7abe74C79f86F0D1212107692430eef72
-FORCED_INCLUSION_STORE=0x05d88855361808fA1d7fc28084Ef3fCa191c4e03
+TAIKO_ANCHOR_ADDRESS=0x1670000000000000000000000000000000010001
 
 # P2P
 DISABLE_P2P_SYNC=false
 P2P_SYNC_URL=https://rpc.mainnet.taiko.xyz
 
+## Driver configs for preconfirmations
+P2P_LISTEN_TCP_PORT=4001
+P2P_UDP_PORT=30303
+# IP address where your node can be reached from at port 4001
```

Comment:
add comment for UDP port, "port used for discv5 discovery on the p2p network,.must be open and external and accessible via the same IP address as the public IP var